README.md

```
@@ -1034,9 +1049,33 @@ The project includes installer scripts for easy deployment:
 
 ## Development
 
+### Prerequisites
+
+- Rust 1.70 or later (https://rustup.rs/)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)